src/main/java/fr/lunatech/timekeeper/models/time/TimeSheet.java

```diff
@@ -64,9 +64,20 @@
     @OneToMany(mappedBy = "timeSheet")
     public List<TimeEntry> entries;
 
+    @Null
```

Comment:
Should it be @NotNull ? Because the startDate is required, and it is set to the creation date by default. 

See [TK-446]( https://lunatech.atlassian.net/jira/software/projects/TK/boards/8?selectedIssue=TK-446) -> 

> By default, the start date must be the date of creation